### DISCUSSION TITLE
Puppet

### DISCUSSION THREAD

**Discussion by Innerlight2111** [Security-Onion-Solutions/securityonion#7425] [Unsupported Versions] [answered]
I am a content creator for my organization and we commoly have to spin up a Security Onion cluster for training. For our other VMs like windows servers and other hosts we use puppet. I was wondering if there was any work being done to create some sort of automation for the installer. I know the installer sets up salt and automates the installation, what I'm asking more specifically is a way to automate the installer.

**Comment by dougburks** [comment, marked as answer]
Our Professional Services team can help with automation. Please see https://securityonionsolutions.com/support for more information and to reach out to them.

**Reply by Innerlight2111** [reply]
Thank you Doug, I will have to find out what our contract is with your company to see what support options I have. Do you have an e-mail that I can send you a more private message as you will have more information about my organization. Thank you again.
